tests/classes/monograph/SubmissionFileDAOTest.php

```diff
@@ -85,17 +87,14 @@ protected function setUp(): void
         $monograph->setData('locale', 'en');
         $submissionDao->expects($this->any())
             ->method('get')
-            ->will($this->returnValue($monograph));
+            ->willReturn($monograph);
         Repo::submission()->dao = $submissionDao;
 
         // Register a mock genre DAO.
         $genreDao = $this->getMockBuilder(GenreDAO::class)
             ->onlyMethods(['getById'])
             ->getMock();
         DAORegistry::registerDAO('GenreDAO', $genreDao);
-        $genreDao->expects($this->any())
-            ->method('getById')
-            ->will($this->returnCallback([$this, 'getTestGenre']));
```

Comment:
@asmecher this seems to break the tests as there is no existence of method `getTestGenre` . is this a leftover of some old test refactoring ?

Reply:
Yes, it looks like it got removed with https://github.com/pkp/omp/commit/36ee1af3200b5811d637bd948be5df939ed9ec70 but this reference persisted afterwards.